docs/pages/application-access/okta/reference.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Okta Service Reference Documentation
+description: Configuration and CLI reference documentation for Teleport Okta service.
+---
+
+## Configuration
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+The following snippet shows the full YAML configuration of an Okta Service
+appearing in the `teleport.yaml` configuration file:
```

Comment:
```suggestion
The following snippet shows the full YAML configuration of the Okta Service
as it would appear in the `teleport.yaml` configuration file:
```